.github/CODEOWNERS

```diff
@@ -13,6 +13,7 @@ yarn.lock                                         @backstage/maintainers @backst
 /docs/features/techdocs                           @backstage/techdocs-maintainers
 /docs/plugins/integrating-search-into-plugins.md  @backstage/discoverability-maintainers
 /packages/cli/src/commands/onboard                @backstage/sharks
+/packages/backend-openapi-utils                   @backstage/maintainers @backstage/openapi-tooling-maintainers
```

Comment:
Could you add @backstage/reviewers as well? 🙏

Reply:
done!